crates/collab/src/tests/test_server.rs

```diff
@@ -273,7 +273,7 @@ impl TestServer {
             collab_ui::init(&app_state, cx);
             file_finder::init(cx);
             menu::init();
-            settings::KeymapFile::load_asset("keymaps/default.json", cx).unwrap();
+            settings::KeymapFile::load_asset("keymaps/default-macos.json", cx).unwrap();
```

Comment:
Missed this version of the keyamp filename the first time around.

I'm leaving all the tests as using the macos keymap for now. Long term it would probably make more sense for them to use the platform default, but in the interest of not breaking macos this feels like the right move for now. Either way it's hard to imagine the keymap used matters.